cmd/manager/main.go

```diff
@@ -134,15 +135,16 @@ func main() {
 		watch.SelectNamespacesPredicate(config.WatchedNamespaces), // select only desired namespaces
 	}
 
+	atlasManager := atlas.NewCloudFactory(config.AtlasDomain, config.GlobalAPISecret, mgr.GetClient())
```

Comment:
I think this is more descriptive:
```suggestion
	atlasCloud := atlas.NewProductionCloud(config.AtlasDomain, config.GlobalAPISecret, mgr.GetClient())
```